docs/tutorials/nav2-integration.md

```diff
@@ -1,3 +1,107 @@
 # Using Beluga with Nav2
 
-Coming soon!
+:::{figure} ../_images/beluga_nav2.gif
+:alt: Short video of the integration of beluga_amcl with nav2.
+:::
+
+## Overview
+
+[Nav2](https://docs.nav2.org/index.html) is a ROS2 framework that includes several components for robot navigation. Among these components is the localization system, which typically uses the [nav2_amcl](https://github.com/ros-navigation/navigation2/tree/humble/nav2_amcl) implementation.
+
+This tutorial will demonstrate how to integrate [beluga_amcl](../packages/beluga_amcl/docs/index.md) into the Nav2 stack as an alternative to the default `nav2_amcl`.
+
+To keep things straightforward, we’ll base this tutorial on the [Nav2 Getting Started Guide](https://docs.nav2.org/getting_started/index.html#). We will use the simulated robot and environment provided by [TurtleBot 3](https://emanual.robotis.com/docs/en/platform/turtlebot3/quick-start/).
+
+## Requirements
+
+:::{important}
+This tutorial is using ROS2 **Humble**.
+:::
+
+Install [ROS2](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html), [Beluga](../getting-started/installation.md) and [Nav2](https://docs.nav2.org/getting_started/index.html#installation).
+
+## Create the workspace
+
+Since we are using ROS2, we need to create a workspace. Open a terminal and run the following command:
+
+```bash
+mkdir -p ~/beluga_demo_ws/src
+```
+
+## beluga_demo_nav2 package
+
+### Clone beluga_demo
+
+The [beluga_demo](https://github.com/Ekumen-OS/beluga-demos) is a public repository where you can find the `beluga_demo_nav2` package, among others.
+
+Clone the repository into the workspace you created earlier:
+
+```bash
+cd ~/beluga_demo_ws/src
+git clone https://github.com/Ekumen-OS/beluga-demos.git
+```
+
+:::{note}
+For this tutorial, we will use only the `beluga_demo_nav2` package from the `beluga_demo` repository. This package contains all the necessary files to launch the simulation.
+:::
+
+### Explanation
+
+As explained in the [beluga_amcl ROS2 reference](../packages/beluga_amcl/docs/ros2-reference.md), `beluga_amcl` is feature-wise compatible with `nav2_amcl`.
+
+Because of this, we copied the necessary folders and files from `nav2_bringup` to the `beluga_demo_nav2` package, simplified them, and replaced the use of `nav2_amcl` with `beluga_amcl` in the `localization_launch.py` launch file, as shown in the following images.
```

Comment:
@Alondruck nit: 
```suggestion
Since `beluga_amcl` is feature-wise compatible with `nav2_amcl` (as thoroughly discussed in the [`beluga_amcl` ROS 2 reference page](../packages/beluga_amcl/docs/ros2-reference.md), we can simply adapt `nav2_bringup` to use `beluga_amcl` instead of `nav2_amcl`, modifying the `localization_launch.py` launch file:
```

Reply:
We should test the link locally and make sure it works.